contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -588,7 +592,72 @@ <h3>Self-certification</h3>
 
 </body>
 
+<script src="https://cdn.jsdelivr.net/npm/@tarekraafat/autocomplete.js@9.0.0/dist/js/autoComplete.min.js"></script>
 <script>
+
+  $( function() {
+    function split( val ) {return val.split( /,\s*/ );}
+    function extractLast( term ) {return split( term ).pop();}
+
+    $( "#contribution_Admins" )
+        .on( "keydown", function( event ) {
+        if ( event.keyCode === $.ui.keyCode.TAB &&
+            $( this ).autocomplete( "instance" ).menu.active ) {
+          event.preventDefault();
+        }
+      })
+      .autocomplete({
+        source: function(request, response) {
+            $.ajax({
+                url: "https://api.github.com/search/users",
+                data: {
+                    q: extractLast(request.term)
+                },
+                type: "GET",
+                headers: {Authorization: 'token ' + '{{token}}'},
+                success: function(data) {
+                        var res = [];
+                        data.items.forEach((e) => res.push(e.login));
+                        console.log(res);
```

Comment:
@wenjzhu, please delete console log statements in this PR. Please also remove commented code. The rest looks fine. Thanks.

Reply:
updated it